pkg/kubectl/cmd/label.go

```diff
@@ -57,6 +57,9 @@ $ kubectl label pods foo bar-`
 )
 
 func NewCmdLabel(f *cmdutil.Factory, out io.Writer) *cobra.Command {
+	p := kubectl.NewHumanReadablePrinter(false, false, false, false, []string{})
```

Comment:
@smarterclayton I'm not sure if I understand it... `NewHumanReadablePrinter` is used here to retrieve all resource types (pod, service, etc.) that we handle, which should be the same for kubectl no matter which printer is used. Can you please clarify that for me?